torchsparse/nn/modules/conv.py

```diff
@@ -51,14 +59,16 @@ def __init__(self,
             assert not transpose
 
     def __repr__(self):
+        _stride = str(self.stride) if isinstance(self.stride, int) else str(
+            self.stride[0].cpu().numpy().tolist())
         if not self.t:
-            return 'Conv3d(in_channels=%d, out_channels=%d, kernel_size=%d, stride=%d, dilation=%d)' % (
-                self.in_channels, self.out_channels, self.kernel_size,
-                self.stride, self.dilation)
+            return 'Conv3d(in_channels=%d, out_channels=%d, kernel_size=%s, stride=%s, dilation=%d)' % (
+                self.in_channels, self.out_channels, str(
+                    self.kernel_size), _stride, self.dilation)
         else:
-            return 'Conv3d(in_channels=%d, out_channels=%d, kernel_size=%d, stride=%d, dilation=%d)' % (
-                self.in_channels, self.out_channels, self.kernel_size,
-                self.stride, self.dilation)
+            return 'Conv3d(in_channels=%d, out_channels=%d, kernel_size=%s, stride=%s, dilation=%d)' % (
+                self.in_channels, self.out_channels, str(
+                    self.kernel_size), _stride, self.dilation)
```

Comment:
We might need to add an additional field called `transpose` to indicated whether it's a deconvolution.